docs/metrics.asciidoc

```diff
@@ -333,12 +334,13 @@ The reason for that is that if you set your sampling rate to 10%, for example,
 you'll only be storing one out of 10 requests.
 The labels you set on non-sampled transactions will be lost.
 
-
 [float]
 [[metrics-micrometer-beta-caveats]]
-==== Beta caveats
-* To fix some of the caveats listed here, we may have to introduce breaking changes.
-  So please watch release notes Micrometer-related changes before updating.
+==== Caveats
+
+To fix some of the caveats listed here, we may have to introduce breaking changes.
+Please look for Micrometer-related release notes before updating.
+
 * Metrics with dots in their name can result in mapping conflicts.
 See https://github.com/elastic/apm/issues/347[elastic/apm#347]
 * Histograms aren't supported yet
```

Comment:
It's not completely accurate that Histograms are not supported. Histograms are partially supported creating two counters that already enable insightful information: a sum counter (xxx.sum) and a count counter (xxx.count).

Instead of the explanation "Histograms aren’t supported yet", an explanation that would look like


> Histograms ([`DistributionSummary`](https://www.javadoc.io/doc/io.micrometer/micrometer-core/latest/io/micrometer/core/instrument/DistributionSummary.html), [`Timer`](https://www.javadoc.io/doc/io.micrometer/micrometer-core/latest/io/micrometer/core/instrument/Timer.html)and [`LongTaskTimer`](https://www.javadoc.io/doc/io.micrometer/micrometer-core/latest/io/micrometer/core/instrument/LongTaskTimer.html)) are partially supported converting the histogram metric into two derived metric: a counter of the values and the sum of the values.
> For example, `DistributionSummary.builder("order").register(...).record(orderPrice)` will create two metrics: `order.sum`and `order.count`.
 
TODO: we should verify that `Timer` and `LongTaskTimer` behave the same way as `DistributionSummary` creating these two derived metrics.